msvc2013/GiGi/GiGi.vcxproj

```diff
@@ -230,7 +232,7 @@
       </LinkStatus>
       <TreatLinkerWarningAsErrors>
       </TreatLinkerWarningAsErrors>
-      <AdditionalDependencies>opengl32.lib;glu32.lib;glew32.lib;freetype255MT.lib;libpng16.lib;User32.lib</AdditionalDependencies>
+      <AdditionalDependencies>opengl32.lib;glu32.lib;glew32.lib;freetype255MT.lib;libpng16.lib;User32.lib;SDL2.lib</AdditionalDependencies>
```

Comment:
Adding SDL as a dependency to GiGi itself is probably not a good idea. It's supposed to be usable with various window managers, such as SDL or Ogre.

It can be added to GiGiSDL to replace the default GiGi-internal implementation similar to what I had before (or lack thereof), though.